test/module/irohad/ametsuchi/postgres_block_storage_test.cpp

```diff
@@ -100,6 +106,7 @@ class PostgresBlockStorageTest : public ::testing::Test {
   std::string dbname_ = generator::randomString(8);
   std::string pgopt_ = "dbname=" + dbname_ + " "
       + integration_framework::getPostgresCredsOrDefault();
+  PostgresOptions options_{pgopt_, "testdb", storage_logger_};
```

Comment:
please use a named constant or `integration_framework::getRandomDbName()` for database name